tool_collections/kraken/kraken/kraken.xml

```diff
@@ -9,7 +9,23 @@
     <command>
         <![CDATA[
         kraken --threads \${GALAXY_SLOTS:-1} @INPUT_DATABASE@ 
+        
+        #if isinstance( $input_sequences.datatype, $__app__.datatypes_registry.get_datatype_by_extension('fastq').__class__):
```

Comment:
Added docs for this https://github.com/galaxyproject/planemo/commit/d42cd7d77734c6b485b61d415ca4a3d60f01edf9